### PR TITLE
📖  docs: eks docs refactor and added eks console docs

### LINKED ISSUE #2186: Docs on how to use EKS Console with CAPA
/kind feature
/area provider/eks
/kind documentation

**Describe the solution you'd like**
I would like some documentation on how I can use the EKS console with clusters created by CAPA.

**Environment:**

- Cluster-api-provider-aws version:  0.6.3

### PR DESCRIPTION
**Pull request by richardcase**
**What this PR does / why we need it**:
This adds documentation on how to use the EKS console with clusters created with CAPA. Also the EKS docs have been reorganised.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2186

### CODE REVIEW

File: docs/book/src/topics/eks/creating-a-cluster.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+To get the user kubeconfig for a cluster named `managed-test` you can run a command similar to:
+
+```bash
+kubectl --namespace=default get secret managed-test-user-kubeconfig \
+   -o jsonpath={.data.value} | base64 --decode \
+   > managed-test.kubeconfig
+```
+
+### CAPI kubeconfig
```
Comment (randomvariable) [inline docs/book/src/topics/eks/creating-a-cluster.md:33]:
```suggestion
### Cluster API (CAPI) kubeconfig
```

Expand the acronym for the first use.

File: docs/book/src/topics/eks/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- **Feature status:** Experimental
+- **Feature gate (required):** EKS=true
+- **Feature gate (optional):** EKSEnableIAM=true,EKSAllowAddRoles=true
+
+## Overview
+
+Experimental support for EKS has been introduced in the AWS provider. Currently the following features are supported:
+
+- Provisioning/managing an AWS EKS Cluster
```
Comment (randomvariable) [inline docs/book/src/topics/eks/index.md:11]:
```suggestion
- Provisioning/managing an Amazon EKS Cluster
```
nit: using the official name

File: docs/book/src/topics/eks/creating-a-cluster.md (hunk 1 above)
Comment (richardcase) [reply docs/book/src/topics/eks/creating-a-cluster.md]: Good idea, changed.

File: docs/book/src/topics/eks/index.md (hunk 2 above)
Comment (richardcase) [reply docs/book/src/topics/eks/index.md]: I like it, changed

**Comment by richardcase** [comment]
@michaelbeaumont @randomvariable  - would you be able to take a look at this please?

**Pull request merged**